src/Microsoft.ML.Transforms/CategoricalTransform.cs

```diff
@@ -171,16 +171,14 @@ public CategoricalTransform(TermEstimator term, IEstimator<ITransformer> keyToVe
 
         public bool IsRowToRowMapper => _transformer.IsRowToRowMapper;
 
-        public IRowToRowMapper GetRowToRowMapper(ISchema inputSchema)
-        {
-            Contracts.CheckValue(inputSchema, nameof(inputSchema));
```

Comment:
> Contrac [](start = 11, length = 8)

did you intend to omit the check?

Reply:
As @Zruty0 pointed out we run check inside _transformer.GetRowToRowMapper

---
In reply to: [221369314](https://github.com/dotnet/machinelearning/pull/1033#discussion_r221369314) [](ancestors = 221369314)